app/controllers/application_controller.rb

```diff
@@ -33,10 +33,16 @@ class ApplicationController < ActionController::Base
   end
 
   rescue_from Exceptions::UnauthorizedBulkRequest do |exception|
-    #return head :unauthorized # for use only with JSON requests
+    # for use only with JSON requests
     render json: { errors: ['title' => exception.message] }, status: 401
   end
 
+  rescue_from Exceptions::MergedEntityError do |e|
+    EntitiesController::TABS.include?(action_name) ?
```

Comment:
help me out here -- where does this rescue_from block get the binding for ` action_name `

Reply:
just a magic rails function you can call in any controller (`some_controller.action_name`) to retrieve the action that is currently being handled. see (eg):

http://guides.rubyonrails.org/action_controller_overview.html#routing-parameters